aws-lambda-scorer-python/terraform-recipe/api_gateway.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_api_gateway_rest_api" "scorer_api" {
```

Comment:
Let's find a way to reuse these instead of copy pasting this from the Java part. Now if we need to change anything, we have to go to two places and keep them in sync. Also it is not clear what the actual difference is this way. Seems to me it is only the `handler` and `runtime`.
Maybe we could use the existing terraform recipe and turn it into a module: https://blog.gruntwork.io/how-to-create-reusable-infrastructure-with-terraform-modules-25526d65f73d.

Reply:
Planned for next iteration. Will need to change code on the other template to include a variables.tf, this would mean some changes to the ci as well.

Reply:
It's fine (or even preferred) to do it separately, but let's do it before this one is merged then. I would rather not have copy&pasted code around unless strictly needed. Is that ok?